beetmoverscript/docker.d/worker.yml

```diff
@@ -127,6 +141,14 @@ bucket_config:
           buckets:
             nightly_components: 'maven-nightly-stage-s3-upload-bucket'
         dummy: {}
+        dep:
```

Comment:
We're likely getting real credentials only next week but I think for the dev workers, we can totally fake with dummy values in SOPS until then.

Reply:
We did, I think we're good here.